### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.3][Eclipse][Wipro]:Keyboard focus order is illogical in  deploy as web app window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1373]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app.
4. In "Deploy web app" dialog will open. 
5. In "Deploy web app" dialog , select any existing app service and tab up to  the "Show FTP Deployment Credentials" link then press tab and observe the keyboard focus order.

**By using Shift+Tab(Backward Navigation):**

1. Create a new dynamic web project.
2. Navigate to and select Project ->Azure->Publish as Azure web app.
3. In "Deploy web app" dialog will open. 
4. In "Deploy web app" dialog, tab up to cancel button.
5. From Cancel button, Press Shift+Tab then observe the keyboard focus order

**Expected Result:**
**In forward Navigation:**
Keyboard focus order should be appservice(present under Name)->Create->Delete->Refresh->App Service Details Pane->links present in App Service details pane->Deploy to root checkbox->Deploy->Cancel
**In Backward Navigations(Using Shift+Tab):**
Cancel->Deploy->Deploy to root checkbox->links present in App Service details pane->App Service Details Pane->Refresh->Delete->Create->App service(Present under Name Column)

**Actual Result:**
**In forward Navigation:**
Keyboard focus order is appservice(present under Name)->Create->Delete->Refresh->App Service Details Pane->links present in App Service details pane->Deploy->Cancel
->> In Forward Navigation keyboard focus is not moving to deploy to root checkbox 
**In Backward Navigations(Using Shift+Tab):**
Keyboard focus order is  Cancel->Deploy->Deploy to root checkbox->App Service Details Pane->links present in App Service details pane->Delete->Create-> App Service(under name column)
->>In Backward Navigation keyboard focus is not moving to Refresh button

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version**: Version 1709(Os Build 16299.192)

![keyboard focus is not logical using tab and shift tab in deploy web app](https://user-images.githubusercontent.com/22956130/35680847-7eea7018-0781-11e8-9983-1f8cb5b7b3c2.jpg)

**Comment by jdneo** [comment]
Caused by the `Browser` widget behavior.

Same behavior in Eclipse's official Brwoser page:
![image](https://user-images.githubusercontent.com/6193897/38720067-68913a76-3f27-11e8-9410-ebdcc6be5288.png)

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533978

**Comment by Eskibear** [comment]
linkded to #1619

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry.
* **Quality of experience**: not impacted. Users can view the app service details in Azure Portal. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

See:
![image](https://user-images.githubusercontent.com/2351748/46596756-2c6c2400-cb10-11e8-84cd-6646a0264979.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749816
